docs/authentication/fb-google-oauth2.html

```diff
@@ -1,28 +1,360 @@
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+    </div>
+    <div class="docs-item-2 search-container">
+      <!-- <input class="docs-search" type="text" name="searchInput" placeholder="Search documentation..."> -->
+      <div id="beta" class="temp-beta">
+        <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
+      </div>
+    </div>
+    <nav class="docs-item-3 docs-nav">
+      <button id="api-button" class="apiref-button" type="button" name="button" onclick="window.open('https://ibm-swift.github.io/Kitura/')">API Reference</button>
+      <a class="nav-item" target="_blank" href="http://slack.kitura.io/">Need help?</a>
+      <a class="nav-item" target="_blank" href="https://github.com/IBM-Swift/kitura.io/issues">Found an issue?</a>
+    </nav>
+    <div id="doc-container" class="docs-item-4 docs-window">
+      <main>
+  <h1 class="heading-1">OAuth2 authentication with Facebook/Google </h1>
+  <p class="block-text">OAuth2 is an authorization framework that enables you to authenticate a user via a trusted third party.
+      A brief summary of the protocol flow is as follows:<p>
+  <ol>
+      <li>The user wants to log in to a server (e.g. your Kitura application).</li>
+      <li>The user is redirected to the OAuth2 provider (e.g. Facebook/Google).</li>
+      <li>The user authenticates with the OAuth2 provider and is given a single use access code.</li>
+      <li>The user is sent back to the server and provides them with the access code.</li>
+      <li>The server sends the access code to the OAuth2 provider and is given an access token.</li>
+      <li>The server uses the access token to request the user's information and authenticate them.</li>
+  </ol>
+  <p>If you would like a more in-depth explanation of OAuth2, you can read <a target="_blank" href="https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2">this guide</a> by DigitalOcean.</p>
+  <p class="block-text">Kitura supports OAuth2 authentication using plugins for the <a target="_blank" href="https://github.com/IBM-Swift/Kitura-Credentials">Kitura-Credentials</a> authentication framework.
+       In this guide, we show you how to use the <a target="_blank" href="https://github.com/IBM-Swift/Kitura-CredentialsGoogle">Kitura-CredentialsGoogle</a> and the <a target="_blank" href="https://github.com/IBM-Swift/Kitura-CredentialsFacebook">Kitura-CredentialsFacebook</a> plugins.
+       Because we are authenticating using a third party, the user does not need to create a new account for our service and we don't need to handle their username and password.</p>
+  <div class="info">
+  <p class="block-text">This guide describes how to implement the OAuth2 flow for a client accessing the server via a web browser.
+       For a mobile application, the user must authenticate themselves on the device, send the access token to the server and have the token authenticated using a Credentials token plugin.</p>
```

Comment:
I wondered if we could separate these two sentences with a blank line in the block box and maybe add "which we're not going to cover in this guide." or similar to the end of the second sentence just to make it clear that we're not covering the second case.